docs/source/traits_api_reference/trait_handlers.rst

```diff
@@ -11,13 +11,11 @@ Classes
 
 .. autoclass:: TraitCastType
 
-.. autoclass:: ThisClass
-
 .. autoclass:: TraitInstance
 
-.. autoclass:: TraitFunction
+.. autodata:: TraitFunction
```

Comment:
Please can you convert these back to `autoclass`?

Reply:
Thanks for pointing out, converted in 0b5a698.